### PR TITLE
ui: hide triggered by and status filters for now

### PR DESCRIPTION
**Pull request by lgfa29**
Using query param filters with the new `filter` param will return an error, so hiding these for now.

### CODE REVIEW

File: ui/app/templates/evaluations/index.hbs [hunk 1]
```diff
@@ -12,6 +12,7 @@
         />
       </div>
       <div class="toolbar-item is-right-aligned">
+        <!-- TODO: add support for status and triggered by
```
Comment (philrenaud) [inline ui/app/templates/evaluations/index.hbs:15]: `{{!--` as a comment style in .hbs will prevent it from showing up in rendered output -- either way is totally fine, but if we don't want this rendered in built source, you could switch your `<` for a `{{` etc.

**Review by philrenaud — APPROVED** [review]

File: ui/app/templates/evaluations/index.hbs (hunk 1 above)
Comment (lgfa29) [reply ui/app/templates/evaluations/index.hbs]: Yeah, thanks! The linter screamed at me for using the wrong one 😄

**Comment by mikenomitch** [comment]
Also needs the Type filter to be commented out

**Review by ChaiWithJai — APPROVED** [review]

**Comment by lgfa29** [comment]
> Also needs the Type filter to be commented out - this one breaks it too

Ah yeah, thanks!

**Review by mikenomitch — APPROVED** [review]

**Pull request merged**